analytics/run-rbqm-analytics.R

```diff
@@ -0,0 +1,75 @@
+# TODO:
+# - tweak data to introduce greater outliers
+# - add explanatory comments to script and to workflows
+
+library(purrr)
+library(dplyr)
+library(gsm)
+
+# 0. Input data
+# ----
+
+lRawData <- list(
+    dm = readRDS('data/SDTM/dm.rds'),
+    ds = readRDS('data/SDTM/ds.rds'),
+    lb = readRDS('data/SDTM/lb.rds'),
+    ae = readRDS('data/SDTM/ae.rds')
+)
+
+# 1. Mapped data
+# ----
+
+lMappingWorkflows <- gsm::MakeWorkflowList(
+    strPath = 'workflows/1_mappings',
+    strPackage = NULL
+)
+
+lMappedData <- gsm::RunWorkflows(
+    lMappingWorkflows,
+    lRawData
+)
+
+# 2. Analysis Data
+# ----
+
+lAnalysisWorkflows <- gsm::MakeWorkflowList(
+    strPath = 'workflows/2_metrics',
+    strPackage = NULL
+)
```

Comment:
I can understand showing them how to build their own mapping workflows, but shouldn't we use gsm.kri for this step (and gsm.reporting for step 3, then back to gsm.kri for step 4)? Once the data is mapped to the expected format, the downstream workflows should work as-is, right? If we teach people to create their own workflows for everything (including set reports), they'll inevitably break any time we change any small thing.